File: search_logos.js
const fs = require('fs');
const path = require('path');

const directory = 'C:/Users/HP/.gemini/antigravity/scratch/downloaded/chunks';
const keywords = ['ACM UMT', 'Logo/', 'Gamer Lounge', 'Cyber Security'];

if (!fs.existsSync(directory)) {
  console.log('Directory not found:', directory);
  process.exit(1);
}

const files = fs.readdirSync(directory);
for (const file of files) {
  if (!file.endsWith('.js')) continue;
  const filePath = path.join(directory, file);
  const content = fs.readFileSync(filePath, 'utf8');
  
  for (const kw of keywords) {
    if (content.includes(kw)) {
      console.log(`Keyword '${kw}' found in file '${file}':`);
      const idx = content.indexOf(kw);
      console.log(content.substring(Math.max(0, idx - 200), Math.min(content.length, idx + 500)).replace(/\n/g, ' '));
      console.log('-'.repeat(40));
    }
  }
}

File: search_logos.py
import os

directory = "C:/Users/HP/.gemini/antigravity/brain/ae26e466-18ba-4bdc-9bf4-f593dbc88cd9/scratch/chunks"
output_file = "C:/Users/HP/.gemini/antigravity/scratch/techverse-clone/search_results.txt"
keywords = ["ACM UMT", "Logo/", "Gamer Lounge", "Cyber Security", "gdgocumt"]

results = []

if os.path.exists(directory):
    for fn in os.listdir(directory):
        if not fn.endswith(".js"):
            continue
        path = os.path.join(directory, fn)
        with open(path, "r", encoding="utf-8", errors="ignore") as f:
            content = f.read()
        
        for kw in keywords:
            if kw in content:
                results.append(f"Keyword '{kw}' in file '{fn}':")
                idx = content.find(kw)
                snippet = content[max(0, idx-250):min(len(content), idx+650)].replace("\n", " ")
                results.append(snippet)
                results.append("-" * 50)
else:
    results.append(f"Directory not found: {directory}")

with open(output_file, "w", encoding="utf-8") as f:
    f.write("\n".join(results))
print("Done writing results")
